src/components/SingleName/Address/constants.js

```diff
@@ -13,41 +13,53 @@ const COIN_INFO = {
     name: 'Litecoin',
     encoding: 'base58check'
   },
-  // ETH: {
-  //   index: 60,
-  //   symbol: 'ETH',
-  //   base: 16,
-  //   name: 'Ethereum',
-  //   encoding: 'hex-checksum'
-  // },
+  ETH: {
```

Comment:
@Arachnid @jefflau coin info exists on both `ens-app` and `ui` which seem redundant. Can it be consolidated into `address-encoder` ? Also, do we need base and encoding info?